examples/tour.jl

```diff
@@ -71,26 +71,26 @@ for s in [Tables.Schema((:a, :b, :c, :d), (Real, String, Any, AbstractVector)),
     # if `complies_with` finds a violation, it returns `false`; returns `true` otherwise
     @test complies_with(s, FooV1SchemaVersion())
 
-    # if `validate` finds a violation, it throws an error indicating the violation;
+    # if `validate` finds any violations, it throws an error indicating the violations;
     # returns `nothing` otherwise
     @test validate(s, FooV1SchemaVersion()) isa Nothing
 
-    # if `find_violation` finds a violation, it returns a tuple indicating the relevant
-    # field and its violation; returns `nothing` otherwise
-    @test isnothing(find_violation(s, FooV1SchemaVersion()))
+    # if `find_violations` finds any violations, it returns a vector of tuples indicating the relevant
+    # fields and their violations; returns `nothing` otherwise
+    @test isnothing(find_violations(s, FooV1SchemaVersion()))
```

Comment:
I think the new `find_violations` function should always return a vector:
```suggestion
    # if `find_violations` finds any violations, it returns a vector of tuples indicating the relevant
    # fields and their violations
    @test isempty(find_violations(s, FooV1SchemaVersion()))
```
I made this change in https://github.com/beacon-biosignals/Legolas.jl/pull/87 along with another change if you want to pull it in